### ISSUE TITLE
Mermaid does not support return type in a classDiagram

### ISSUE THREAD

**Issue by LeoCHANGcn** [BoostIO/Boostnote#3542]
# Current behavior

0.15.2
win10 1909

```
classDiagram
class BankAccount{
    +String owner
    +BigDecimal balance

    +deposit(amount) bool
    +withdrawl(amount) int
}
```

Get a result:
![image](https://user-images.githubusercontent.com/25730864/78519927-58489280-77f7-11ea-97ba-0947589650f6.png)

But it should be:

![image](https://user-images.githubusercontent.com/25730864/78519938-60083700-77f7-11ea-8259-48158a5a49c9.png)

**Comment by Flexo013** [comment]
This is indeed rendered wrong. See https://mermaid-js.github.io/mermaid/#/classDiagram?id=return-type

My first guess is that return types are supported yet by our current version of mermaid, but I couldn't easily find when this feature was introduced.

**Comment by Flexo013** [comment]
This is solved as of Boostnote Legacy version 0.16.0.

Thanks for your contribution @daiyam ❤️
